API.md

```diff
@@ -22,3 +22,8 @@ This page documents the SDX Gateway API endpoints. Apart from the Service Inform
 **Required parameters**: `caseId` as the ID of the case the response receipt is for.
 
 An `HTTP 201 Created` status code is returned if the response receipt creation was a success. An `HTTP 400 Bad Request` is returned if the required parameter is missing.
+
+## Acknowledge File
+* `POST /paperquestionnairereceipts/multipart/form-data`
```

Comment:
From the looks of it, it's just `/paperquestionnairereceipts`, the multipart form-data doesn't look like it's included in the url, I think it's just the required Content-Type that needs to be in the HTTP header... but don't quote me on that ^^